### PR TITLE
First version v0.1.0 of argo-apps-viz

### PR DESCRIPTION
**Pull request by Syndlex**

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/argo-apps-viz.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - from: LICENSE
+      to: "."
+    bin: "argo-apps-viz.exe"
+  homepage: https://github.com/syndlex/argo-apps-viz
+  caveats: |
+    Usage:
+      $ kubectl argo-apps-viz
+    For additional options:
+      $ kubectl argo-apps-viz --help
+      or https://github.com/syndlex/argo-apps-viz/blob/v0.1.0/doc/USAGE.md
```
Comment (ahmetb) [inline plugins/argo-apps-viz.yaml:50]: We print this already after plugin installation. Consider deleting.

File: plugins/argo-apps-viz.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  shortDescription: A kubectl plugin that visualizes ArgoCD concepts like the 'App of Apps' and dependency trees for documentation and teaching.
```
Comment (ahmetb) [inline plugins/argo-apps-viz.yaml:51]: Please cap at 55 chars. Things like
"A kubectl plugin that" are very obvious in this context already.

File: plugins/argo-apps-viz.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  description: |
+    shortDescription: A small kubectl plugin that generates visual representations (using go-echarts) to document and illustrate ArgoCD concepts, such as the 'App of Apps' pattern across multiple clusters and dependency trees, primarily for documentation, learning, and teaching purposes.
```
Comment (ahmetb) [inline plugins/argo-apps-viz.yaml:53]: Please wrap at 80 char lines.

**Comment by Syndlex** [comment]
@ahmetb thank you for your corrections. Was to tired yesterday sorry for your inconvenience.

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!
Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Pull request merged**